packages/cf.js/src/client.ts

```diff
@@ -0,0 +1,87 @@
+import * as uuid from "uuid";
+
+import { AppFactory } from "./app-factory";
+import { AppInstance } from "./app-instance";
+import {
+  AppDefinition,
+  MessageDataQuery,
+  NodeMessage,
+  NodeMessageType,
+  NodeProvider,
+  QueryType
+} from "./structs";
+
+export enum ClientEventType {
```

Comment:
These are moreso "Request"s not "ClientEvent"s?

Reply:
No. There are events that aren't request-response e.g. your counterparty proposes to install an app.